controllers/release/adapter.go

```diff
@@ -180,14 +181,58 @@ func (a *adapter) EnsureReleaseIsRunning() (controller.OperationResult, error) {
 	return controller.ContinueProcessing()
 }
 
+// EnsureTenantReleasePipelineIsProcessed is an operation that will ensure that a Tenant Release PipelineRun associated to the Release
```

Comment:
doesn't match function name (I like this better than the function name)

Reply:
fixed.